Sources/Contour/Transport/RabbitMQ/Internal/RabbitBus.cs

```diff
@@ -499,62 +410,45 @@ private void StartTask(CancellationToken cancellationToken)
             this.ComponentTracker.StartAll();
 
             this.logger.Trace(m => m("{0}: marking as ready.", this.Endpoint));
-            IsStarted = true;
-            ready.Set();
+            this.IsStarted = true;
+            this.ready.Set();
```

Comment:
Is there any need to have two state sync (flags) objects to control the start/stop behavior? It would be more consistent to have the IsStarted property return the state of a ManualResetEventSlim property IsSet (if at all).
In an ideal case imho there should be only one state sync object.